### PR TITLE
chore(pipelined): Get patched aioeventlet from fork

### LINKED ISSUE #13425: Create and use a fork of aioeventlet
**Subtasks**

- [x] Set up Magma fork of aioeventlet
- [x] Commit patches, readme and notice information to new repository
- [x] Create PR 
- [x] Merge it
- [x] Prepare internal review meeting

**Description:**

Instead of patching the existing library, which will not be possible for Python > 3.8, we want to publish a Magma fork that already contains the changes. Finish the fork in:
https://github.com/sebathomas/deb-python-aioeventlet/

Use the fork everywhere in Magma. Remove the patch from the Magma repo because it is already contained in the fork. Proof of concept for that:
https://github.com/magma/magma/pull/13044

### PR DESCRIPTION
**Pull request by sebathomas**
## Summary
Instead of patching the unmaintained official version of `aioeventlet` in various build and deploy steps (issue was discussed in #6368), the PR applies the patched version of `aioeventlet` from [Magma fork](https://github.com/magma/deb-python-aioeventlet).

## Test Plan

- [x] Unittests
- [x] Sudo Tests
- [x] `make run`
- [x] `sudo service magma@* status` running
- [x] LTE integration tests
- [x] Containerized AGW

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: third_party/build/bin/aioeventlet_build.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ PKGVERSION=0.5.1
 ITERATION=2
 VERSION="$PKGVERSION-$ITERATION"
 PKGNAME=python3-aioeventlet
-REPO="https://github.com/openstack-archive/deb-python-aioeventlet.git"
```
Comment (maxhbr) [inline third_party/build/bin/aioeventlet_build.sh:22]: no pinning here?

**Comment by mpfirrmann** [comment]
Results from testing the `aioeventlet` from the patched fork:

- [ ] Unittests
- [ ] Sudo Tests
- [ ] LTE integration tests
- [ ] `make run`

File: third_party/build/bin/aioeventlet_build.sh (hunk 1 above)
Comment (sebathomas) [reply third_party/build/bin/aioeventlet_build.sh:22]: Good point, I think most of the x_build.sh get the code from master at the moment.

But I added pinning here.

**Review by Neudrino — APPROVED** [review]
~~This will also likely break the builds for all old branches (if they are not currently broken).~~

~~Should we add some documentation somewhere to make this change known?~~

File: bazel/python_repositories.bzl [hunk 2]
```diff
@@ -37,15 +37,10 @@ def python_repositories(name = ""):
...
         build_file = "//bazel/external:aioeventlet.BUILD",
-        commit = "0afed1425fd36139d1a7281475a28fde3349047e",  # Corresponds to: tag = "0.5.1"
-        shallow_since = "1456158219 +0100",
-        patches = [
-            "//lte/gateway/deploy/roles/magma/files/patches:aioeventlet_fd_exception.patch",
-            "//lte/gateway/deploy/roles/magma/files/patches:aioeventlet.py38.patch",
-        ],
-        patch_args = ["-p1"],
+        commit = "86130360db113430370ed6c64d42aee3b47cd619",
+        shallow_since = "1655813728",
```
Comment (Neudrino) [inline bazel/python_repositories.bzl:43]: Should this actually be the current datetime or the point of the last commit in the repository?
Do we need it at all, as we do not have (or plan to have) too many branches in this repository? After all it is blow 100KB in size.

**Comment by sebathomas** [comment]
> warning This will also likely break the builds for all old branches (if they are not currently broken).
> 
> Should we add some documentation somewhere to make this change known?

Not sure I follow. Why does this affect old branches?

**Comment by Neudrino** [comment]
> Not sure I follow. Why does this affect old branches?

You are right, as we do not change infrastructure here, there should be no problems as we still have all the old artifacts and build scripts. :sweat_smile:

**Review by crasu — APPROVED** [review]

File: bazel/python_repositories.bzl (hunk 2 above)
Comment (sebathomas) [reply bazel/python_repositories.bzl:43]: I think it's just an optimization so the point in time needs to be before the latest commit. Maybe a good question for bazel experts.

File: bazel/python_repositories.bzl [hunk 3]
```diff
@@ -37,15 +37,10 @@ def python_repositories(name = ""):
     )
     new_git_repository(
         name = "aioeventlet_repo",
-        remote = "https://github.com/openstack-archive/deb-python-aioeventlet.git",
+        remote = "https://github.com/sebathomas/deb-python-aioeventlet.git",
```
Comment (sebathomas) [inline bazel/python_repositories.bzl]: Oops, this needs to be magma, not sebathomas.

File: bazel/python_repositories.bzl (hunk 2 above)
Comment (sebathomas) [reply bazel/python_repositories.bzl:43]: As discussed with Nils offline, we may want to change the aioeventlet setup to Bazel in the future, see #13599.

**Review by voisey — APPROVED** [review]

**Review by nstng — APPROVED** [review]
lgtm

**Comment by voisey** [comment]
Hi @magma/approvers-agw, could someone please take a look at this if you get the chance? Thanks!

**Review by ardzoht — APPROVED** [review]
lgtm

**Pull request merged**